src/system.ts

```diff
@@ -216,6 +216,11 @@ export const arweave = new Arweave({
   protocol: arweaveURL.protocol.replace(':', ''),
 });
 
+export const walletAddress =
+  walletJwk !== undefined
+    ? await arweave.wallets.jwkToAddress(walletJwk)
+    : 'undefined';
```

Comment:
```suggestion
    : undefined;
```

Reply:
but we may not want to omit the field so it might be worth setting as `INVALID` or `NOT_SET`

Reply:
I like `INVALID` (covers a wider range of possibilities).

Reply:
Agreed! It was set as “undefined” on purpose to not omit the field, but INVALID is way better

Reply:
[`4fa218e` (#30)](https://github.com/ar-io/ar-io-observer/pull/30/commits/4fa218eb4001f59452214829d44b8238a0240bf3)